make.jl

```diff
@@ -1,15 +1,64 @@
+import Pkg
+Pkg.activate(".")
+
 using Documenter, DocumenterMarkdown, Turing, AdvancedHMC, Bijectors, AdvancedMH
 using LibGit2: clone, tag_list, GitRepo
+using Documenter: GitHubActions
 
 # Include the utility functions.
 include("make-utils.jl")
 
-# Find the package directory
+# Get the version number
+trim_version(x) = x[1:findlast('.', x) - 1]
+
+version, is_dev = if haskey(ENV, "TURING_VERSION")
+    ENV["TURING_VERSION"], true
+else
+    if length(ARGS) > 0
+        trim_version(ARGS[1]), false
+    else
+        "dev", true
+    end
+end
+
+# Make a temporary folder to build from
+tmp_path = mktempdir()
+
+# Paths
+## The location of the package to build (Turing())
 package_directory = dirname(dirname(pathof(Turing)))
+## The location of the docs folder inside that package
+docs_path = joinpath(package_directory, "docs")
+## The src files for the docs -- markdown documents, typically.
 source_path = joinpath(package_directory, "docs", "src")
+## The path of turing.ml that we are running this code from.
+local_path = @__DIR__
+## The place to put the files from source_path after they go through Documenter.jl
+build_path = joinpath(tmp_path, "_docs")
+
+# Get any files from Turing's directory
+for (root, dirs, files) in walkdir(docs_path)
+    new_root = replace(root, docs_path => tmp_path)
+
+    for file in files
+        old_file = joinpath(root, file)
+        new_file = joinpath(new_root, file)
+        @debug "" old_file new_file
+        if !isdir(dirname(new_file))
+            mkpath(dirname(new_file))
+        end
+        cp(old_file, new_file, force=true)
+    end
+end
 
-# Paths.
-build_path = joinpath(@__DIR__, "_docs")
+# Copy all the local files to the temporary path
+paths = readdir(local_path, join=true)
+filter!(x -> !(basename(x) in ["make.jl", "make-utils.jl"]), paths)
+for path in paths
+    new_path = replace(path, local_path => tmp_path)
+    @debug "" path new_path
+    cp(path, new_path, force=true)
```

Comment:
I think the force=true should not be necessary here as you copy to a new temporary dir and thus the file cannot exists.

Reply:
Good point, fixed.